code/datums/status_effects/debuffs.dm

```diff
@@ -213,6 +220,23 @@
 /datum/status_effect/bluespace_slowdown/on_remove()
 	owner.next_move_modifier /= 2
 
+/datum/status_effect/katana_curse //tracking if user is dead
+	id = "katana_curse"
+	duration = -1
+	alert_type = null
+	var/deathTick = 0
```

Comment:
couldn't you use a signal for this instead? register a signal listening for `COMSIG_MOB_DEATH` when you implant the organ and remove it when you remove the organ.

Reply:
Signal was mentioned on discord, did not quite work right, and would not pass cl due to sleeps in CL

Reply:
signal now working